pkg/pebinaryexecutor/pebinary.go

```diff
@@ -173,12 +155,9 @@ func (p *PEBinaryConfig) Kubelet(ctx context.Context, args []string) error {
 // KubeProxy starts the kubeproxy in a subprocess with watching goroutine.
 func (p *PEBinaryConfig) KubeProxy(ctx context.Context, args []string) error {
 	extraArgs := map[string]string{
-		"hostname-override": p.cniConig.CalicoConfig.Hostname,
-		"v":                 "4",
-		"proxy-mode":        "kernelspace",
-		"kubeconfig":        p.cniConig.NodeConfig.AgentConfig.KubeConfigKubeProxy,
-		"network-name":      p.cniConig.NetworkName,
-		"bind-address":      p.cniConig.BindAddress,
+		"v":            "4",
```

Comment:
We don't normally override the log level for components, is there some particular reason to ship this with a higher-than-normal log verbosity?